cellprofiler/modules/graytocolor.py

```diff
@@ -182,25 +201,23 @@ def add_stack_channel_cb(self, can_remove=True):
             len(self.stack_channels) % len(DEFAULT_COLORS)]
         group.append("image_name", cps.ImageNameSubscriber(
                 "Image name", cps.NONE,
-                doc=
-                """
-                <i>Used only if %(SCHEME_STACK)s or %(SCHEME_COMPOSITE)s is
-                chosen</i><br>
-                Select the input image to add to the stacked image""" % globals()))
+                doc='''\
+*Used only if %(SCHEME_STACK)s or %(SCHEME_COMPOSITE)s is chosen*
+Select the input image to add to the stacked image
+''' % globals()))
         group.append("color", cps.Color(
                 "Color", default_color,
-                doc="""
-            <i>Used only if %(SCHEME_COMPOSITE)s is chosen</i>
-            <br>The color to be assigned to the above image.
-            """ % globals()))
+                doc='''\
+*Used only if %(SCHEME_COMPOSITE)s is chosen*
+The color to be assigned to the above image.
+''' % globals()))
         group.append("weight", cps.Float(
                 "Weight", 1.0, minval=.5 / 255,
-                doc="""
-            <i>Used only if %(SCHEME_COMPOSITE)s is chosen</i>
-            <br>The weighting of the above image relative to the others. The
-            image's pixel values are multiplied by this weight before assigning
-            the color.
-            """ % globals()))
+                doc='''\
+*Used only if %(SCHEME_COMPOSITE)s is chosen*
```

Comment:
Add parentheses around this first sentence.

Add a blank line between *(Used only if...)* and the next line.